src/java.base/share/classes/sun/net/www/ParseUtil.java

```diff
@@ -177,9 +177,9 @@ public static String decode(String s) {
         StringBuilder sb = new StringBuilder(n);
         ByteBuffer bb = ByteBuffer.allocate(n);
         CharBuffer cb = CharBuffer.allocate(n);
-        CharsetDecoder dec = ThreadLocalCoders.decoderFor(UTF_8.INSTANCE)
-            .onMalformedInput(CodingErrorAction.REPORT)
-            .onUnmappableCharacter(CodingErrorAction.REPORT);
+        CharsetDecoder dec = UTF_8.INSTANCE.newDecoder()
+                .onMalformedInput(CodingErrorAction.REPLACE)
```

Comment:
This changes the error action from REPORT to REPLACE. This has implications for all usages of ParseUtil. Did you mean to do this?

Reply:
Good catch Alan. I overlooked this. This should not be changed.

Reply:
That's an error, thanks for catching.